map/bookmark_manager.hpp

```diff
@@ -377,6 +380,13 @@ class BookmarkManager final
 
   bool HasRecentlyDeletedBookmark() const { return m_recentlyDeletedBookmark.operator bool(); };
   void ResetRecentlyDeletedBookmark();
+  
+  size_t GetRecentlyDeletedCategoriesCount() const;
+  BookmarkManager::KMLDataCollectionPtr GetRecentlyDeletedCategories();
```

Comment:
Can it be const?

Reply:
nit: an alternative approach may leverage something like `ForEachDeletedCategory(TLambda &&)`, but let's leave it for a separate discussion later.

Reply:
> Can it be const?

No, because the LoadBookmarks cannot.

Reply:
Actually, LoadBookmarks method should be marked as const, because it doesn't modify any members. Then GetRecentlyDeletedCategories() can also be properly marked as const. Otherwise, reading it, there is an assumption that it changes the internal state, while it's not.